chain/jsonrpc/src/lib.rs

```diff
@@ -250,9 +250,12 @@ impl JsonRpcHandler {
 
     async fn tx_status_fetch(
         &self,
-        tx_hash: CryptoHash,
-        account_id: AccountId,
+        tx_info: TransactionInfo,
```

Comment:
Wouldn't it make sense to have `tx_info` a borrowed TransactionInfo instead of owned? (This will avoid unnecessary clones on the call sight)

Reply:
good point

Reply:
actually it seems that the cloning at call sight cannot be avoided